database/migrations/2015_11_04_015813_create_link_table.php

```diff
@@ -29,6 +29,8 @@ public function up()
             $table->boolean('is_disabled')->default(0);
             $table->boolean('is_custom')->default(0);
             $table->boolean('is_api')->default(0);
+            $table->boolean('is_expried')->default(0);
+            $table->date('expried_date')->nullable();
```

Comment:
I think this should be `expiry_date` or `expires_at`

Reply:
Also, this should be in a new migration so existing users can get the functionality